pkg/build/controller/build/defaults/api/types.go

```diff
@@ -46,8 +46,7 @@ type BuildDefaultsConfig struct {
 	// resources defines resource requirements to execute the build.
 	Resources kapi.ResourceRequirements
 
-	// tolerations is a list of default Tolerations that are applied to build pods
-	// so that they can be placed on build-specific nodes
+	// tolerations is a list of default Tolerations that are appended to build pods
```

Comment:
are appended to the build pod's set of tolerations.

Reply:
updated in current push